src/test/java/core/basesyntax/service/impl/WriterServiceImplTest.java

```diff
@@ -0,0 +1,87 @@
+package core.basesyntax.service.impl;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.fail;
+
+import core.basesyntax.service.WriterService;
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.Collections;
+import java.util.List;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+public class WriterServiceImplTest {
+    private static final String CORRECT_PATH_TO_FILE = "src/main/resources/output.csv";

```

Comment:
```suggestion
    private static final String CORRECT_PATH = "src/main/resources/output.csv";

```

Reply:
And you should make tests for FruitTransaction and all classes in operation pakcage.